CommonFindPackage.cmake

```diff
@@ -269,4 +269,8 @@ macro(common_find_package_post)
   endif()
   common_graph(${PROJECT_NAME})
   message(STATUS ${__configure_msg})
+
+  # If this variable was given in the command line, ensure that the package
+  # installation is only run in this cmake invocation.
+  unset(INSTALL_PACKAGES CACHE)
```

Comment:
I don't think this can be moved to SubProject.cmake because it has to be run just once and after all subprojects have been processed. This seems to be the only place where this can be guaranteed.
I thought about it, but if you are not using CommonFindPackage correctly this is the smallest of your problems. On the other hand, does it make sense to use this project if you're not going to use common_find_package? If you are cherry picking files you can take care of this details yourself as well.